### ISSUE TITLE
nan isiue

### ISSUE THREAD

**Issue by devlakshmi** [AlexeyAB/darknet#4709]

**Comment by AlexeyAB** [comment]
Try to train on GPU.

**Comment by devlakshmi** [comment]
@AlexeyAB  currently I don't have a GPU

**Comment by WongKinYiu** [comment]
it seems the objects in your dataset are very big (larger than 100x100 i think).

**Comment by devlakshmi** [comment]
> it seems the objects in your dataset are very big (larger than 100x100 i think).
> 
> only 82th layer has value:
> ![image](https://user-images.githubusercontent.com/12152972/72534402-b4afac00-38b1-11ea-9c56-e5cb29c651e3.png)
> 
> anchors of 82th layer are 116x90, 156x198, and 373x326.
> https://github.com/AlexeyAB/darknet/blob/master/cfg/yolov3.cfg#L607-L615

@WongKinYiu  the anchors are there , is there any reason why its happening

**Comment by ramdhan1989** [comment]
similar problem I have. shown below :
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 130 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: nan, .5R: -nan, .75R: -nan, count: 0, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 82 Avg (IOU: 0.000000, GIOU: 0.000000), Class: -nan, Obj: -nan, No Obj: -nan, .5R: 0.000000, .75R: 0.000000, count: 1, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 94 Avg (IOU: 0.000000, GIOU: 0.000000), Class: -nan, Obj: nan, No Obj: nan, .5R: 0.000000, .75R: 0.000000, count: 3, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 106 Avg (IOU: 0.000000, GIOU: 0.000000), Class: -nan, Obj: nan, No Obj: nan, .5R: 0.000000, .75R: 0.000000, count: 5, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 118 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: nan, .5R: -nan, .75R: -nan, count: 0, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 130 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: nan, .5R: -nan, .75R: -nan, count: 0, loss = -nan, class_loss = -nan, iou_loss = -nan

 (next mAP calculation at 1000 iterations) 
 7: -nan, -nan avg loss, 0.000000 rate, 3060.289851 seconds, 448 images
Loaded: 0.000159 seconds
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 82 Avg (IOU: 0.000000, GIOU: 0.000000), Class: -nan, Obj: nan, No Obj: -nan, .5R: 0.000000, .75R: 0.000000, count: 7, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 94 Avg (IOU: 0.000000, GIOU: 0.000000), Class: -nan, Obj: nan, No Obj: nan, .5R: 0.000000, .75R: 0.000000, count: 2, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 106 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: nan, .5R: -nan, .75R: -nan, count: 0, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 118 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: nan, .5R: -nan, .75R: -nan, count: 0, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 130 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: nan, .5R: -nan, .75R: -nan, count: 0, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 82 Avg (IOU: 0.000000, GIOU: 0.000000), Class: -nan, Obj: -nan, No Obj: -nan, .5R: 0.000000, .75R: 0.000000, count: 6, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 94 Avg (IOU: 0.000000, GIOU: 0.000000), Class: -nan, Obj: nan, No Obj: nan, .5R: 0.000000, .75R: 0.000000, count: 7, loss = -nan, class_loss = -nan, iou_loss = -nan
v3 (mse loss, Normalizer: (iou: 0.750000, cls: 1.000000) Region 106 Avg (IOU: -nan, GIOU: -nan), Class: -nan, Obj: -nan, No Obj: nan, .5R: -nan, .75R: -nan, count: 0, loss = -nan, class_loss = -nan, iou_loss = -nan
